src/udev/udevadm-trigger.c

```diff
@@ -39,18 +39,17 @@ static int exec_list(sd_device_enumerator *e, const char *action, Set *settle_se
                 if (!filename)
                         return log_oom();
 
-                fd = open(filename, O_WRONLY|O_CLOEXEC);
-                if (fd < 0)
+                r = write_string_file(filename, action, WRITE_STRING_FILE_CREATE);
```

Comment:
WRITE_STRING_FILE_CREATE looks wrong, there was no O_CREAT in the original open() line. That said, I think WRITE_STRING_FILE_DISABLE_BUFFER is a good thing for all writes to kernel sysfs files and such

Reply:
OK. So, the same for files in /proc ? I noticed that there are several calls for /proc that do not have the flag.

Reply:
yeah, we probably should indeed do that...